packages/components/package.json

```diff
@@ -11,19 +11,18 @@
   ],
   "scripts": {
     "build": "stencil build",
-    "dev": "stencil build --dev --watch --serve",
+    "dev": "stencil build --dev --watch",
```

Comment:
run `serve` on the project root and open `localhost:5000`. This dont have autoreload but every change will be reflected on manual reload.

Reply:
autoreload is so nice though :[

Reply:
yeah, but is no easy to get it running and since our last implementation was buggy I prefer to remove it. We can open an issue to implement it properly :)